PowerEditor/src/WinControls/TaskList/TaskList.cpp

```diff
@@ -205,18 +211,19 @@ LRESULT TaskList::runProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
 			else
 			{
 				int32_t selected = (_currentIndex + 1) > (_nbItem - 1) ? 0 : (_currentIndex + 1);
-				ListView_SetItemState(_hSelf, _currentIndex, 0, LVIS_SELECTED|LVIS_FOCUSED);
+				ListView_SetItemState(_hSelf, _currentIndex, 0, LVIS_SELECTED|LVIS_FOCUSED);				

```

Comment:
This is nice PR. So I would say, better you refresh the PR by removed reformatting and make sure it remains in single commit. You can use 
amend -a    to amend with previous  and
push -f    to force rewrite